src/Test/Utilities/Portable/FX/DumpProcRunner.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities
+{
+    public class ProcDumpRunner
+    {
+        public const string ProcDumpPathEnvironmentVariableKey = "ProcDumpPath";
+
+        private const string ProcDumpExeFileName = "procdump.exe";
+
+        // /accepteula command line option to automatically accept the Sysinternals license agreement.
+        // -ma	Write a 'Full' dump file. Includes All the Image, Mapped and Private memory.
+        // -e	Write a dump when the process encounters an unhandled exception. Include the 1 to create dump on first chance exceptions.
+        // -h	Write dump if process has a hung window (does not respond to window messages for at least 5 seconds).
```

Comment:
Also it matches with [http://source.roslyn.io/#ProcessWatchdog/ProcDump.cs,74](http://source.roslyn.io/#ProcessWatchdog/ProcDump.cs,74)

---
In reply to: [122541539](https://github.com/dotnet/roslyn/pull/20129#discussion_r122541539) [](ancestors = 122541539,122364809)